README.md

```diff
@@ -95,6 +94,14 @@ How do folks in the community use RFC 4122 UUIDs?
   it available in the standard library eventually saves TBs of bandwidth globally. If we continue
   to address user needs, such as `uuid`, with the standard library, bandwidth savings add up.
 
+**Why does the standard library API treat `v4` UUIDs as a default instead of being symmetric in the
+different versions?**
+
+An analysis of popular Open Source projects that were using `v1` UUIDs has shown, that the majority
+of identified projects was "accidentally" using `v1` UUIDs in situations where `v4` UUIDs would
```

Comment:
`was accidentally` -> `were accidentally` ... I might instead word it as,

> did not have a compelling reason for using v1, and with education were willing to migrate.

I think we should then plan to reach out to these folks and put some hard numbers around this hypothesis, e.g., we reached out to 10 projects and all but 2 were willing to migrate.